### DISCUSSION TITLE
Remaining work for 1.0?

### DISCUSSION THREAD

**Discussion by xldenis** [zesterer/chumsky#642] [General]
Hi, 

Is there a list of tasks that contributors could tackle to help bring 1.0 closer to reality? I would love to help make this possible.

**Comment by zesterer** [comment]
Hello! There is a tracking issue here: #543. I'm mostly lacking time/energy to push it forward right now. A lot of it is docs work, but there's also some more abstract thinking around certain API choices. I'd appreciate input/help, let me know if there's anything I can explain!

**Reply by xldenis** [reply]
Do you know if the issue is accurate / up to date? If it is, I might try to knock off some of the smaller chore jobs next week.

**Reply by zesterer** [reply]
Sorry about the delayed response, life has been busy. It is broadly accurate, yes. If you feel like working on something, let me know and I can give guidance / context / help / etc.!
